addons/l10n_it_edi/tests/expected_xmls/IT00470550013_discounted_fully.xml

```diff
@@ -0,0 +1,90 @@
+  <FatturaElettronicaHeader>
+    <DatiTrasmissione>
+      <IdTrasmittente>
+        <IdPaese>IT</IdPaese>
+        <IdCodice>01234560157</IdCodice>
+      </IdTrasmittente>
+      <ProgressivoInvio>2022030003</ProgressivoInvio>
+      <FormatoTrasmissione>FPR12</FormatoTrasmissione>
+      <CodiceDestinatario>0000000</CodiceDestinatario>
+      <ContattiTrasmittente>
+      </ContattiTrasmittente>
+    </DatiTrasmissione>
+    <CedentePrestatore>
+      <DatiAnagrafici>
+        <IdFiscaleIVA>
+          <IdPaese>IT</IdPaese>
+          <IdCodice>01234560157</IdCodice>
+        </IdFiscaleIVA>
+        <CodiceFiscale>01234560157</CodiceFiscale>
+        <Anagrafica>
+          <Denominazione>company_2_data</Denominazione>
+        </Anagrafica>
+      </DatiAnagrafici>
+      <Sede>
+        <Indirizzo> </Indirizzo>
+        <CAP>00000</CAP>
+      </Sede>
+    </CedentePrestatore>
+    <CessionarioCommittente>
+      <DatiAnagrafici>
+        <IdFiscaleIVA>
+          <IdPaese>IT</IdPaese>
+          <IdCodice>00465840031</IdCodice>
+        </IdFiscaleIVA>
+        <Anagrafica>
+          <Nome>Alessi</Nome>
+          <Cognome></Cognome>
+        </Anagrafica>
+      </DatiAnagrafici>
+      <Sede>
+        <Indirizzo>Via Privata Alessi 6 </Indirizzo>
+        <CAP>28887</CAP>
+        <Nazione>IT</Nazione>
+      </Sede>
+    </CessionarioCommittente>
+  </FatturaElettronicaHeader>
+  <FatturaElettronicaBody>
+    <DatiGenerali>
+      <DatiGeneraliDocumento>
+        <TipoDocumento>TD01</TipoDocumento>
+        <Divisa>EUR</Divisa>
+        <Data>2022-03-24</Data>
+        <Numero>INV/2022/03/0003</Numero>
+      </DatiGeneraliDocumento>
+    </DatiGenerali>
+    <DatiBeniServizi>
+      <DettaglioLinee>
+        <NumeroLinea>1</NumeroLinea>
+        <Descrizione>nothing shady just a gift for my friend</Descrizione>
+        <Quantita>1.00</Quantita>
+        <PrezzoUnitario>800.400000</PrezzoUnitario>
+        <ScontoMaggiorazione>
+          <Tipo>SC</Tipo>
+          <Percentuale>100.00</Percentuale>
+        </ScontoMaggiorazione>
+        <PrezzoTotale>0.00</PrezzoTotale>
+        <AliquotaIVA>22.00</AliquotaIVA>
+      </DettaglioLinee>
+      <DatiRiepilogo>
+        <AliquotaIVA>22.00</AliquotaIVA>
+        <ImponibileImporto>0.00</ImponibileImporto>
+        <Imposta>0.00</Imposta>
+        <EsigibilitaIVA>I</EsigibilitaIVA>
+      </DatiRiepilogo>
+    </DatiBeniServizi>
+    <DatiPagamento>
+      <CondizioniPagamento>TP02</CondizioniPagamento>
+      <DettaglioPagamento>
+        <DataScadenzaPagamento>2022-03-24</DataScadenzaPagamento>
+        <ImportoPagamento>0.00</ImportoPagamento>
+        <CodicePagamento>INV/2022/03/0003</CodicePagamento>
+      </DettaglioPagamento>
+    </DatiPagamento>
+    <Allegati>
```

Comment:
What is the point in having the attachments here in the code when we do not use them to compare?